CHANGELOG.md

```diff
@@ -28,7 +28,8 @@ release.
 </tr>
 <tr>
   <td valign="top">
-    <b><a href="doc/changelogs/CHANGELOG_V7.md#7.5.0">7.5.0</a></b><br/>
+    <b><a href="doc/changelogs/CHANGELOG_V7.md#7.6.0">7.6.0</a></b><br/>
+    <a href="doc/changelogs/CHANGELOG_V7.md#7.4.0">7.5.0</a><br/>
```

Comment:
I don't think it's worth a rebuild. I'd rather just have 7.6 today. The link gets you close enough 😄